src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -108,16 +108,17 @@ const ApplicationModal = ({
   useEffect(() => {
     const getCurrentApp = () => {
       Object.values(applicationProfile).map(current_id => {
-        if (current_id?.profile_id == profileId)
```

Comment:
iterating over object values was a problem for me at some point, good use of the Object built-in methods for iteration